### ISSUE TITLE
el-get recipes incorrectly omit some files

### ISSUE THREAD

**Issue by dadinn** [radian-software/straight.el#1009]
## What's wrong

trying to install/load [systemd-mode](https://github.com/holomorph/systemd-mode) fails, with error `Error (use-package): Cannot load systemd-mode`.

Looking at the *straight-byte-compilation* buffer I can see the following logs:

```
Compiling /home/dadinn/.emacs.d/straight/build/systemd-mode/systemd.el...

In toplevel form:
systemd.el:93:22:Error: Opening input file: No such file or directory, /home/dadinn/.emacs.d/straight/build/systemd-mode/unit-directives.txt
Done (Total of 0 files compiled, 1 failed, 1 skipped)
```

## Directions to reproduce

`(straight-use-package 'systemd-mode)`

### Version information

* Emacs version: 27.1
* Operating system: Debian 11

**Comment by progfolio** [comment]
By passing the symbol `systemd-mode` you are getting the el-get recipe:

https://github.com/dimitri/el-get/blob/84dd1837f9ac80a329ab0c2de6859777f445f8ff/recipes/systemd-mode.rcp

Which does not include a `:files` directive to link the package's text files.
You can either add the files directive:

```emacs-lisp
(systemd-mode :files (:defaults "*.txt"))
```

Or install by requesting the symbol `systemd`, which is how MELPA advertises the package:

https://github.com/melpa/melpa/blob/master/recipes/systemd

The MELPA recipe has the correct `:files` directive, so you would not need to add it manually in that case.

```emacs-lisp
(straight-use-package 'systemd)
```

**Comment by dadinn** [comment]
Raised the issue over there at the el-get repo too: https://github.com/dimitri/el-get/issues/2884

**Comment by npostavs** [comment]
Hello, I realize this is quite old, but I'm looking at some open issues in el-get, and I don't see this as an el-get bug.

el-get doesn't require specifying `:files` to get the .txt files, so it seems to me that straight is misinterpreting el-get's recipe (which implicitly takes all the files). In fact, el-get doesn't support a `:files` directive in its recipes at all.

**Comment by raxod502** [comment]
Oh, okay. Does el-get take _all_ the files, or are there some default exclusions like in MELPA?

**Comment by npostavs** [comment]
Yes, el-get takes all the files; it just does a full checkout (or tarball unpack, etc depending on `:type`).
